### ISSUE TITLE
On select event?

### ISSUE THREAD

**Issue by kgrosvenor** [zurb/tribute#292]
How to get the active data inside tribute other then text or spans, i'd expect a array of tags and the raw text?

**How can we reproduce this bug?**

1. Step one
2. Step two
3. Step three

**What did you expect to happen?**

**What happened instead?**

**Link (jsfiddle/plunkr/codepen) or Screenshot:**
You can fork this Codepen as a base for reproducing your issue: 
https://codepen.io/mrsweaters/pen/OxxdWv?editors=1010#0

**Comment by jkhaui** [comment]
What do you mean - like a NodeList with each Tribute item's properties when it's active? Or you only want the properties of the item that's selected?

https://codepen.io/jkhaui/pen/BXLMGM?editors=0010

^ Check the console after triggering a tribute menu, I added some console log statements with the active tribute items. But I'm not sure if that's what you were after

**Comment by kgrosvenor** [comment]
Yeah essentially the node list but only and array of string values were interested in plus the text typed including the trigger text

**Comment by kgrosvenor** [comment]
Another event listener I think for

tribute-selected / changed

**Comment by kgrosvenor** [comment]
{
  text: 'hi (Gordon), 
  values: ["Gordon"] 
}

**Comment by kgrosvenor** [comment]
Currently to achieve this behaviour i have

```
getMentions(e: KeyboardEvent) {
                this.inputValue = e.target.innerText;
              
                this.mentionValues = Array.from(this.$el.querySelectorAll('.tribute-mention')) .map(x => x.innerText);

                this.$emit('mentions', {
                    text: this.inputValue,
                    mentions: this.mentionValues
                });

                this.$emit('input', this.inputValue)
            },
```
